hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -669,6 +677,259 @@
         ]
       }
     }
+    //===============================================================
+    // TPM registers
+    { skipto: "0x800"}
+    { name: "TPM_CAP"
```

Comment:
btw, do you think we should add a `constant` type to our regfiles?
For example, the values in this register are never going to change right? But instead of doing a `hwext`, maybe it makes sense to have `constant`  type, then we don't have to declare the flops in side `prim_subreg`

Reply:
Sounds good